doc/contributing/security-release-process.md

```diff
@@ -74,6 +74,17 @@ The current security stewards are documented in the main Node.js
   (Re-PR the pre-approved branch from nodejs-private/nodejs.org-private to
   nodejs/nodejs.org)
 
+  If the security release will only contain an OpenSSL update consider
+  adding the following to the pre-release announcement:
+
+  ```text
+  Since this security release will only include updates for OpenSSL, if you're using
+  a Node.js version which is part of a Linux distribution which uses a system
```

Comment:
```suggestion
  a Node.js version which is part of a distribution which uses a system
```

I can confirm that https://github.com/nodejs/TSC/issues/1187#issuecomment-1067331504 is true. Node.js from Homebrew does dynamically link to the system installed OpenSSL.

<details>

List of shared libraries used by `node` installed using `brew install node`:
```console
$ otool -L $(command -v node)
/usr/local/bin/node:
	/usr/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.11)
	/usr/local/opt/libuv/lib/libuv.1.dylib (compatibility version 2.0.0, current version 2.0.0)
	/usr/local/opt/brotli/lib/libbrotlidec.1.dylib (compatibility version 1.0.0, current version 1.0.9)
	/usr/local/opt/brotli/lib/libbrotlienc.1.dylib (compatibility version 1.0.0, current version 1.0.9)
	/usr/local/opt/c-ares/lib/libcares.2.dylib (compatibility version 2.0.0, current version 2.5.1)
	/usr/local/opt/libnghttp2/lib/libnghttp2.14.dylib (compatibility version 36.0.0, current version 36.2.0)
	/usr/local/opt/openssl@1.1/lib/libcrypto.1.1.dylib (compatibility version 1.1.0, current version 1.1.0)
	/usr/local/opt/openssl@1.1/lib/libssl.1.1.dylib (compatibility version 1.1.0, current version 1.1.0)
	/usr/local/opt/icu4c/lib/libicui18n.70.dylib (compatibility version 70.0.0, current version 70.1.0)
	/usr/local/opt/icu4c/lib/libicuuc.70.dylib (compatibility version 70.0.0, current version 70.1.0)
	/usr/local/opt/icu4c/lib/libicudata.70.dylib (compatibility version 70.0.0, current version 70.1.0)
	/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1856.105.0)
	/usr/lib/libc++.1.dylib (compatibility version 1.0.0, current version 1200.3.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1311.0.0)
```

</details>